20150316-hosting-wp-part-2.md

```diff
@@ -0,0 +1,249 @@
+
+Moving down the file you will see an _http_ block. The first directive to change is the **keepalive_timeout**, which by default is set to 65. The **keepalive_timeout** determines how many seconds a connection to the client should be kept open before it’s closed by Nginx. This directive should be lowered as you don’t want idle connections sitting there for up to 65 seconds if they can be utilised by new clients. I have set mine to `15`.
+
+For security reasons you should uncomment the **server_tokens** directive and ensure it is set to `off`. This will disable emitting the Nginx version number in error messages and response headers.
+
+Underneath **server_tokens** add the **client_max_body_size** directive and set this to the maximum upload size you require in the WordPress Media Library. I chose a value of `64m`.
+
+Further down the _http_ block you will see a section dedicated to Gzip compression. By default, Gzip is enabled but is disabled for Microsoft Internet Explorer 6, however you should tweak these values further for better handling of static files. First, you should uncomment the **gzip_proxied** directive and set it to `any`, which will ensure all proxied request responses are gzipped. Secondly, you should uncomment the **gzip_comp_level** and set it to a value of `2`. This controls the compression level of a response and can have a value in the range of 1 - 9, however setting this too high can have a negative effect on CPU usage. Finally, you should uncomment the **gzip_types** directive, leaving the default values in place. This will ensure that JavaScript, CSS and other file types are gzipped in addition to the HTML file type.
+
+That’s the basic Nginx configuration dealt with, hit _CTRL X_ followed by _Y_ to save the changes. For the changes to take effect you must restart Nginx, however before doing so you should ensure that the configuration file contains no errors by issuing the following command:
+
+```
+sudo nginx -t
+```
+
+If everything looks ok, go ahead and restart Nginx:
+
+```
+sudo service nginx restart
+```
+
+![Restart nginx](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Image-3-Nginx.jpeg)
+
+If everything worked out ok, you should still be able to see the Nginx welcome page when visiting the FQDN in the browser.
+
+## PHP-FPM
+
+Just like Nginx, the Apt-Get repository does contain a PHP repository, however, it isn’t the most up-to-date so we will use one maintained by [Ondřej Surý](https://launchpad.net/~ondrej/+archive/ubuntu/php5-5.6). Add the repo and update the package lists using the following commands:
```

Comment:
> ... the Apt-Get repository does contain a PHP repository, however, it isn’t the most up-to-date ...

Does Apt-Get need capitals?

Not sure "PHP repository" is right, how about "PHP packages"? So it would end up something like ...

"the apt-get repository does contain PHP packages, however, they are not the most up-to-date".